app/views/insured/product_shoppings/show.html.slim

```diff
@@ -209,15 +209,25 @@ javascript:
   }
 
   function filterByPlanTypes(planTypes, dataArray) {
-      var resultsData = [];
-      planTypes.forEach(function (plan) {
-          dataArray.forEach(function (enrollment) {
-              if (enrollment.product.health_plan_kind.toLowerCase() == plan.toLowerCase()) {
-                  resultsData.push(enrollment);
-              }
-          })
-      })
-      return resultsData;
+    var resultsData = [];
+
+    planTypes.forEach(function (plan) {
+      dataArray.forEach(function (enrollment) {
+        var planLower = plan.toLowerCase();
+
+        if (planLower === 'premium_value_plan' && enrollment.is_pvp_eligible) {
+            resultsData.push(enrollment);
+        }
+        else if (planLower === 'standard_plan' && enrollment.product.is_standard_plan) {
+            resultsData.push(enrollment);
+        }
+        else if (enrollment.product.health_plan_kind.toLowerCase() === planLower) {
```

Comment:
If I'm understanding things, this will potentially add duplicates to `resultsData` (e.g,. standard plan that is also an HMO). I see there may be some de-duping function for this data elsewhere. And that is based solely on plan cost. Is that accurate (and sufficient)?